packages/nextjs/app/builders/[0x78F348Dc5De2e9f066FAAa0bfD66d397a9260A43]/page.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import Image from "next/image";
+import { FaGithub, FaXTwitter } from "react-icons/fa6";
+import { EnvelopeIcon, MapPinIcon } from "@heroicons/react/24/outline";
+
+const SandsPage = () => {
+  const socialLinks = [
+    { icon: <FaGithub className="h-6 w-6" />, url: "https://github.com/Tejas-Sands", name: "GitHub" },
+    { icon: <FaXTwitter className="h-6 w-6" />, url: "https://x.com/11cyth", name: "Twitter" },
+  ];
```

Comment:
Since these are constants, it makes sense to place them outside the component. So they're created only once at module initialization, rather than every render.